### PR TITLE
[codestyle] Make anonymous class into named class

### PR DESCRIPTION
**Pull request by Enigmatrix**
Made PrefixIgnoredInlineFilterListener from existing anonymous class

### CODE REVIEW

File: xwiki-rendering-syntaxes/xwiki-rendering-syntax-wikimodel/src/main/java/org/xwiki/rendering/internal/parser/wikimodel/WikiModelParserUtils.java [hunk 1]
```diff
@@ -40,6 +40,32 @@ public void parseInline(StreamParser parser, String content, Listener listener)
     {
         parseInline(parser, content, listener, false);
     }
+    
+    private class PrefixIgnoredInlineFilterListener extends InlineFilterListener {
```
Comment (tmortagne) [inline xwiki-rendering-syntaxes/xwiki-rendering-syntax-wikimodel/src/main/java/org/xwiki/rendering/internal/parser/wikimodel/WikiModelParserUtils.java]: This does not follow XWiki codestyle. The `{` should be on next line.

File: xwiki-rendering-syntaxes/xwiki-rendering-syntax-wikimodel/src/main/java/org/xwiki/rendering/internal/parser/wikimodel/WikiModelParserUtils.java [hunk 2]
```diff
@@ -49,34 +75,8 @@ public void parseInline(StreamParser parser, String content, Listener listener,
...
-                    if (this.foundSpace) {
-                        super.onSpace();
-                    } else {
-                        this.foundSpace = true;
-                    }
-                }
-            };
+            WrappingListener inlineFilterListener = new PrefixIgnoredInlineFilterListener();
             inlineFilterListener.setWrappedListener(listener);
-
```
Comment (tmortagne) [inline xwiki-rendering-syntaxes/xwiki-rendering-syntax-wikimodel/src/main/java/org/xwiki/rendering/internal/parser/wikimodel/WikiModelParserUtils.java]: Why remove that empty line ? I find it more readable to separate `inlineFilterListener` creation from the code using it.

**Comment by Enigmatrix** [comment]
Alrightey, I've fixed the above-mentioned issues

**Comment by tmortagne** [comment]
Thanks for your contribution !

**Comment by Enigmatrix** [comment]
Welcome 😄

**Pull request merged**